### PR TITLE
Adds Alexa and DotNet to list of deprecated SDKs

### PR DESCRIPTION
**Pull request by iAmWillShepherd**
This PR moves the Alexa SDK to a deprecated section and adds the official OS header to the readme. 

📝 [Rendered markdown](https://github.com/OneSignal/sdks/blob/20ec56819ca7192a90d554285138894d4150c90c/README.md)

---

Additionally, I've updated the cross-platform section to include the latest .Net SDK.

📝 [Rendered markdown](https://github.com/OneSignal/sdks/blob/79683c47260f3327e7dd212d3078996b6cfb2972/README.md#cross-platform-sdks)

### CODE REVIEW

**Review by emawby — APPROVED** [review]

**Comment by iAmWillShepherd** [comment]
@emawby, I don't have access to push to this repo. Could you merge?

**Comment by iAmWillShepherd** [comment]
@emawby @fhboswell 

Looks like this was never merged and so my PR diff is screwed. I went ahead and updated the title and description to indicate this. Could one of you merge this please, I can't do it.

**Review by emawby — APPROVED** [review]
Lgtm

**Comment by iAmWillShepherd** [comment]
I don't have privileges to push to this repo, so one of you has to press the button.

**Review by fhboswell — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,70 +1,90 @@
...
+| [OneSignal Expo EAS Plugin](https://github.com/OneSignal/onesignal-expo-plugin)                 | [OneSignal Expo EAS Plugin Setup](https://documentation.onesignal.com/docs/react-native-expo-sdk-setup) |                    |
+| [OneSignal Gradle Plugin](https://github.com/OneSignal/OneSignal-Gradle-Plugin)                 |                                                                                                         |                    |
+| [OneSignal CLI](https://github.com/OneSignal/onesignal-cli)                                     |                                                                                                         |                    |
+
+# Deprecated
+
+| Platform                                                                          | Date               |
+| :-------------------------------------------------------------------------------- | ------------------ |
+| [OneSignal Alexa Plugin](https://github.com/OneSignal/OneSignal-Alexa-Nodejs-SDK) | <INSERT DATE HERE> |
```
Comment (fhboswell) [inline README.md:90]: @iAmWillShepherd Should there be a date here?

Comment (iAmWillShepherd) [reply README.md:90]: The date is for your team to decide. I'm not sure when/if y'all have officially deprecated it yet.

**Pull request merged**